test/e2e/wallet.spec.js

```diff
@@ -0,0 +1,115 @@
+const {
+  Mnemonic,
+} = require('@dashevo/dashcore-lib');
+
+const Dash = require('dash');
+
+const getDAPISeeds = require('../../lib/test/getDAPISeeds');
+
+const createClientWithFundedWallet = require('../../lib/test/createClientWithFundedWallet');
+
+describe('e2e', () => {
+  describe('Wallet', function main() {
+    this.timeout(950000);
+
+    let fundedWallet;
+    let emptyWallet;
+    let duplicateWallet;
+    let mnemonic;
+
+    before(async () => {
+      mnemonic = new Mnemonic();
+      fundedWallet = await createClientWithFundedWallet();
+      emptyWallet = new Dash.Client({
+        wallet: {
+          mnemonic,
+        },
+      });
+    });
+
+    after(async () => {
+      if (fundedWallet) {
+        await fundedWallet.disconnect();
+      }
+
+      if (emptyWallet) {
+        await emptyWallet.disconnect();
+      }
+
+      if (duplicateWallet) {
+        await duplicateWallet.disconnect();
+      }
+    });
+
+    describe('empty wallet', () => {
+      it('should have no transaction at first', async () => {
+        const emptyAccount = await emptyWallet.getAccount();
+
+        expect(emptyAccount.getTransactions()).to.be.empty();
+      });
+
+      it('should receive a transaction when as it has been sent', async () => {
+        const emptyAccount = await emptyWallet.getAccount();
+        const fundedAccount = await fundedWallet.getAccount();
+
+        const tx = await fundedAccount.createTransaction({
+          recipient: {
+            amount: 10,
+            address: emptyAccount.getUnusedAddress(),
+          },
+        });
+
+        await fundedAccount.broadcastTransaction(tx);
+
+        expect(emptyAccount.getTransactions()).to.have.lengthOf(1);
```

Comment:
we need to subscribe to an event here.

Example: https://github.com/dashevo/wallet-lib/blob/feat/subscribeToTransaction/src/utils/fundWallet.js